spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/data/redis/RedisProperties.java

```diff
@@ -188,6 +191,14 @@ public Lettuce getLettuce() {
 		 */
 		private Duration maxWait = Duration.ofMillis(-1);
 
+		/**
+		 * Time of milliseconds to sleep between runs of the idle object evictor thread.
+		 *
+		 * When positive, the idle object evictor thread starts. When non-positive, no
+		 * idle object evictor thread runs.
+		 */
+		private Duration idleEvictPeriod = Duration.ofMillis(-1);
```

Comment:
I think this name should align more closely with the setting that it maps to. Something like `timeBetweenEvictionRuns`, perhaps.

Reply:
I totally agree with your opinion. 👍

Reply:
I totally agree with your opinion. 👍